rbi/stdlib/irb.rbi

```diff
@@ -1523,7 +1523,9 @@ end
 
 class IRB::OutputMethod::NotImplementedError < ::StandardError; end
 
-class IRB::ReidlineInputMethod < ::IRB::InputMethod
+class IRB::ReidlineInputMethod < ::IRB::RelineInputMethod; end
```

Comment:
Force pushed a change to remove the unnecessary `initialize` definition. So the Strip test will be slightly out of date @jez.